src/components/attendee/ProfileOverlay/ProfileOverlay.tsx

```diff
@@ -162,6 +169,16 @@ export const ProfileOverlay: React.FC<ProfileOverlayProps> = ({ profile }) => {
     ]
   );
 
+  const history = useHistory();
+  const { worldSlug, spaceSlug } = useWorldAndSpaceByParams();
```

Comment:
This is a bit heavy. `useWorldAndSpaceByParams` does two requests to DB. 
Yes, the result may usually be cached in the browser so it will be as performant. 

Yet, another hook just for the slugs is already present (even inside `useWorldAndSpaceByParams`) - `useSpaceParams`.

Reply:
you're right. Confused the name of the hook. 🤦🏼‍♀️ 
fixed: https://github.com/sparkletown/sparkle/pull/3003/commits/07ec924f308486ffff85bb746fdba69810c19e56